qa/rpc-tests/test_framework/blockchainhelper.py

```diff
@@ -165,15 +179,19 @@ def init_proof_keys(self, bitcoin_test_framework):
     # but an identifier used to:
     # 1) search for the sidechain in the sidechain map
     # 2) generate the certificate and CSW verification keys
-    def create_sidechain(self, sc_name, sidechain_version, optional_parameters = {}, should_fail = False):
+    def create_sidechain(self, sc_name, creation_arguments, should_fail = False):
         """
-        Creates a sidechain with the optional parameters
-        (or the default ones if not provided),
-        only the name (used to store it locally in a map)
-        and the version are mandatory.
+        Creates a sidechain with the given name
+        (used to store it locally in a map) and
+        the creation arguments provided.
+        Note that among these arguments, only
+        "version" is mandatory.
         """
 
-        sc_input = SidechainCreationInput.from_rpc_args(self, sc_name, sidechain_version, optional_parameters)
+        # "version" is a mandatory argument
+        assert("version" in creation_arguments)
```

Comment:
This check is redundant, as it is performed also inside `SidechainCreationInput.from_rpc_args()`

Reply:
Fix looks good to me!